src/auth/Cargo.toml

```
@@ -24,6 +24,13 @@ repository.workspace = true
 keywords.workspace   = true
 categories.workspace = true
 
+[features]
+default = ["aws_lc_rs"]
+
+aws-lc-rs = ["aws_lc_rs"]
+aws_lc_rs = ["rustls/aws_lc_rs"]
+ring      = []
```

What about having a default, which we do not necessarily name as `aws_lc_rs`. And if the application cares to have something other than our default, they can just disable that feature, and provide their own.

I don't want to have N features for N different crypto providers.

```suggestion
default = ["default-crypto-provider"]

default-crypto-provider = ["rustls/aws_lc_rs"]
```